src/Command/MakerCommand.php

```diff
@@ -39,13 +40,15 @@ final class MakerCommand extends Command
     private $io;
     private $checkDependencies = true;
     private $generator;
+    private $params;
 
-    public function __construct(MakerInterface $maker, FileManager $fileManager, Generator $generator)
+    public function __construct(MakerInterface $maker, FileManager $fileManager, Generator $generator, ParameterBagInterface $params)
```

Comment:
The `ParameterBagInterface` is not needed. If we are going to allow a new `typed_properties` config param, it should be injected into the `PhpCompatUtil::class` instead.